locations/spiders/aubert_fr.py

```diff
@@ -0,0 +1,24 @@
+import scrapy
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class AubertFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "aubert_fr"
+    item_attributes = {"brand": "Aubert", "brand_wikidata": "Q62964657", "extras": Categories.SHOP_BABY_GOODS.value}
+    download_delay = 0.5
```

Comment:
Please don't set `download_delay` in the spider unless we need to make it much slower.

Reply:
No probs, was a copy/paste inclusion from another spider